bots/naughty/rhel-7-4

```diff
@@ -0,0 +1 @@
+rhel-7
```

Comment:
Is this logic the right way around? What happens when 7.5 is released? I would think that right now rhel-7 is rhel-7-4, but rhel-7-4 is more specific.